packages/SwingSet/src/types-external.js

```diff
@@ -350,9 +355,7 @@ export {};
  * incarnation.
  *
  *
- * @typedef {object} CreateVatResults
- * @property {object} root
- * @property {VatAdminFacet} adminNode
+ * @typedef {{ adminNode: Guarded<VatAdminFacet>, root: object }} CreateVatResults
```

Comment:
ouch. that seems hard on devex

Reply:
that's because the JSDoc is currently on the interface and not the implementation. `VatAdminFacet` is a concrete type so it would be better to document its implementation and then the result would just be typeof the actual facet.